config.go

```diff
@@ -73,6 +82,7 @@ type Config[H Hash] struct {
 	VerifyPrepareRequest func(p ConsensusPayload[H]) error
 	// VerifyPrepareResponse performs external PrepareResponse verification and returns nil if it's successful.
 	VerifyPrepareResponse func(p ConsensusPayload[H]) error
+	// TODO: may be VerifyCommit callback should be added for case when AntiMEV extensions are enabled.
```

Comment:
This makes sense because it suddenly becomes more than just a signature.

Reply:
Added.

Reply:
Please, see the https://github.com/nspcc-dev/dbft/pull/118#discussion_r1683173276. We have a different understanding on the purpose of this callback.